types/react-native/index.d.ts

```diff
@@ -4589,7 +4589,7 @@ export interface VirtualizedListWithoutRenderItemProps<ItemT> extends ScrollView
 }
 
 /**
- * @see https://reactnative.dev/docs/listview#props
+ * @see https://reactnative.dev/docs/0.63/listview#props
```

Comment:
These are the types for the latest release so the docs link should match. If `ListView` was removed, we should also remove it from the docs.

Reply:
`ListView` was  removed in latest react-native version,  same as `maskedviewios`, `tabbarios`.
did you mean that should also remove type declaration? (like `export interface ListViewProps extends ScrollViewProps`)
@eps1lon

Reply:
I would say so, yes. It might be worth it to apply the other fixes to the docs for the `v0.63`-version

Reply:
> I would say so, yes. It might be worth it to apply the other fixes to the docs for the `v0.63`-version

so this pr will be closed？

Reply:
> so this pr will be closed？
Doesn’t have to, you could integrate the changes from @eps1lon and add your changes to the types in the respective sub-folder. From what I understand, this seems to be the cleanest solution

Reply:
> > so this pr will be closed？
> 
> 
> 
> Doesn’t have to, you could integrate the changes from @eps1lon and add your changes to the types in the respective sub-folder. From what I understand, this seems to be the cleanest solution

ok will update related changes in 0.63.0 folder